components/meter/meter-radial.js

```diff
@@ -72,7 +74,7 @@ class MeterRadial extends MeterMixin(RtlMixin(LitElement)) {
 			<div
 				class="d2l-meter-radial"
 				role="img"
-				aria-label="${this._ariaLabel(primary, secondary)}">
+				aria-label="${this._ariaLabel(primaryAria, secondaryAria)}">
```

Comment:
Just to confirm: in a screen reader, the presence of `aria-label` is going to take precedence over everything inside? So this would read "image, 5 out of 10, <secondary>" and totally ignore the `5/10` and `<secondary>` rendered inside?

Reply:
I've confirmed this is the case!

Reply:
Yep, labels override everything